Remove downloaded scripts and replace with CDN

rahatzamancse opened the pull request:
I think using an existing CDN than downloading a third-party script locally is a better approach. This makes the local repository smaller.

rahatzamancse committed:
change 3 scripts to cdn

rahatzamancse committed:
change 3 scripts to cdn

rahatzamancse committed:
Merge remote-tracking branch 'origin/cdn' into cdn

rahatzamancse committed:
Merge branch 'cdn'

szpak commented on layouts/partials/head/scripts.html:
```diff
@@ -9,14 +9,11 @@
-{{ $lazysizes := resources.Get "js/lazysizes.min.js" | resources.Fingerprint }}
-<script defer src="{{ $lazysizes.RelPermalink }}"></script>
+<script src="https://cdn.jsdelivr.net/npm/enquire.js@2.1.6/dist/enquire.min.js"></script>
+<script defer src="https://cdn.jsdelivr.net/npm/lazysizes@5.2.2/lazysizes.min.js"></script>
```
It makes it easier to update. Btw, it would be good to use the https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity here, to promote good practices.

(I'm not a maintainer of this plugin, but I plan to reuse your MR for my blog)

rahatzamancse replied:
You can set some sort of variables for the URLs and update them. How is it easier to update local files?

szpak replied:
> It makes it easier to update.

I meant, your change makes it easier to update in the future (just bump the version without changing the - outdated - files in the local repo).

The suggestion with the integrity check is independent.

schneidr commented:
Please leave an option to use the theme without a CDN. CDN are problematic from a privacy point of view, since they allow the CDN to track users across websites.

rahatzamancse committed:
Maintainance updates (#2)

* delicious and google bookmarks discontinued

* delicious and google bookmarks discontinued

* fix: i18n keys

* Social services i18n keys had "line" as a key, fixed accordingly.

* fix: added missing share services

Added "telegram" and "whatsapp" in the list.

* fix: breadcrumb not turning-off when in term list

When breadcrumb is turned-off in config file, it remains available in `/categories/{term}/` and `/tags/{term}/ ` list view.

* fix: breadcrumb not turning-off when set in config

Fixed breadcrumb showing in "Section List" regardless of "false" setting in config params.

* upd: deprecated in Hugo v0.119.0, removed in Hugo v0.125.0

* fix: use hugo.IsServer as .Site.IsServer is deprecated

---------

Co-authored-by: I'M YourOnly.One <1079205+techmagus@users.noreply.github.com>
Co-authored-by: techmagus <techmagus@im.youronly.one>

rahatzamancse committed:
Maintainance: Fixed hugo deprecation

rahatzamancse committed:
Remove the unneeded (auto-generated) resources directory (#3)

Co-authored-by: Yoni <yoni@guardicore.com>
